README.md

```diff
@@ -1,2 +1,98 @@
-# publishing-tutorial-sample-repo
-A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2" 
+# Publishing tutorial sample repo
+A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2"
+
+* [Contributing to this repository](#contributing-to-this-repository)
+
+## Contributing to this repository
+
+### Filing a bug
+
+If you have any suggestions to improve or extend this repository, add a new issue in the repository.
+
+### Prerequisites
+
+* You must have an account on GitHub.
+  [Signing up for a new GitHub account](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
+* You must have registered SSH keys in your GitHub account.
+[Adding a new SSH key to your GitHub account](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
+* You must be a member of the `pantheon-developers` or `pantheon-doc-authors` team in the `redhataccess` group in GitHub.
+* You must be logged in to your account on GitHub.
+
+### Forking the repository
+
+Fork the repository so that you can create and work with branches independently of the `redhataccess/pantheon` repository.
+
+1. In a web browser, navigate to https://github.com/redhat-documentation/publishing-tutorial-sample-repo
+1. Click **Fork**.
+1. Click your user space in GitHub.
+
+### Cloning the repository
+
+After you have forked the repository, you must clone it to your local machine and add the original `redhat-documentation/publishing-tutorial-sample-repo` repository as an upstream remote.
+
+1. From a terminal, clone the repository:
+
+    ```sh
+    $ git clone git@github.com:<user-space>/publishing-tutorial-sample-repo.git
+    ```
+1. Set up `redhat-documentation/publishing-tutorial-sample-repo` as the upstream:
+
+    ```sh
+    $ cd publishing-tutorial-sample-repo
+    $ git remote add upstream git@github.com:redhat-documentation/publishing-tutorial-sample-repo.git
+    ```
+
+### Creating a working branch
+
+Pull requests should have a corresponing issue in the repo. Whenever you work on a new issue, you must create a new working branch based on the latest version of the upstream main branch.
+
+1. Ensure you are on main
+
+    ```sh
+    $ git checkout main
+    ```
+1. Ensure your fork is up to date
+
+    ```sh
+    $ git pull upstream main
+    ```
+1. Create a working branch based on the issue ID. For example for issue #2, you can use "2" or "02" for the branch name:
+
+    ```sh
+    $ git checkout -b 02
+    ```       
+
+### Creating a pull request and completing review
```

Comment:
What do you mean by completing review? My suggestion would be "Creating a pull request and requesting a review" or "Submitting a pull request for review"

Reply:
Fixed.